api/envoy/api/v2/cds.proto

```diff
@@ -645,6 +654,50 @@ message Cluster {
   // outgoing connections made by the cluster. Order matters as the filters are
   // processed sequentially as connection events happen.
   repeated cluster.Filter filters = 40;
+
+  // [#not-implemented-hide:] New mechanism for LB policy configuration.
+  // Used only if the :ref:`lb_policy<envoy_api_field_Cluster.lb_policy>`
+  // field has the value :ref:`LOAD_BALANCING_POLICY_CONFIG
+  // <envoy_api_enum_value_Cluster.LbPolicy.LOAD_BALANCING_POLICY_CONFIG>`.
+  LoadBalancingPolicy load_balancing_policy = 41;
+}
+
+// New Load balancing policy selection.
```

Comment:
`[#not-implemented-hide:] Extensible load balancing policy configuration.` would be better here.

Reply:
Done.